### ISSUE TITLE
Feature request: Unlisted albums

### ISSUE THREAD

**Issue by Rickkwa** [bpatrik/pigallery2#661]
**Is your feature request related to a problem? Please describe.**

Be able to create albums/folders that do not show up on the main page or search, but you can still visit it directly if you have the url.

**Describe the solution you'd like**

A possible implementation is to treat albums/folders' whose name starts with `.` or `_` as unlisted. And of course, this feature will be behind a feature toggle.

**Comment by bpatrik** [comment]
I dont mind having it but I do not have the time to develop it. 

Essentially, it would be a server side change, that would need wether not select from the DB or post filter the selection.

**Comment by fabianus76** [comment]
This feature would be terrific !
I came here to submit the same idea :-)

**Comment by Rickkwa** [comment]
@bpatrik I saw your extension PR (#768) that is in alpha. Very excited for extensions to officially release. Is this feature something that can achieved with the current extension functionality?

**Comment by bpatrik** [comment]
Yeah it should be a 5-10 code in the extension to filter out files during indexing.

**Comment by bpatrik** [comment]
Ah I reread your the FR.
It is not possible at the moment but happy to add this to extension capability.
You will need something like a gallery listing event. Where you filter what goes down to the client (ie remove the one that you want to hide).

Ping me once the extension is more stable but such event handler is missing.

**Comment by bpatrik** [comment]
you can now hide certain photos for certain users: 

<img width="2237" height="569" alt="Image" src="https://github.com/user-attachments/assets/9a2d475f-40d6-4aae-bbae-d0a9cd92f9df" />

This not exactly this but a similar feature.
